sdk/storage/azure-storage-queue/apireview/azure/storage/queue/queue_client.pyi

```diff
@@ -0,0 +1,35 @@
+# Stubs for azure.storage.queue.queue_client (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from ._deserialize import deserialize_queue_creation, deserialize_queue_properties
+from ._generated import AzureQueueStorage
+from ._generated.models import SignedIdentifier, StorageErrorException
+from ._message_encoding import TextXMLDecodePolicy, TextXMLEncodePolicy
+from ._shared.base_client import StorageAccountHostsMixin, parse_connection_str, parse_query
+from ._shared.request_handlers import add_metadata_headers, serialize_iso
+from ._shared.response_handlers import process_storage_error, return_headers_and_deserialized, return_response_headers
+from ._shared_access_signature import QueueSharedAccessSignature
+from .models import AccessPolicy, MessagesPaged, QueueMessage, QueuePermissions, QueueProperties
+from azure.core.paging import ItemPaged
+from datetime import datetime
+from typing import Any, Dict, List, Optional, Union
+
+class QueueClient(StorageAccountHostsMixin):
+    queue_name: Any = ...
+    def __init__(self, queue_url: str, queue: Optional[Union[QueueProperties, str]]=..., credential: Optional[Any]=..., **kwargs: Any) -> None: ...
```

Comment:
We could do the "Blobs treatment" here too for consistency.
- Rename `queue` to `queue_name` and only take str and is required.
- Take only `account_url` and add a `from_url` classmethod.

Reply:
Fixed